.github/workflows/ros.yml

```diff
@@ -0,0 +1,151 @@
+name: ROS
+
+on:
+  push:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  noetic-x86:
+    name: Noetic Ubuntu 20.04
+    runs-on: ubuntu-latest
+    container:
+      image: 337955887028.dkr.ecr.us-east-2.amazonaws.com/kpsr-docker-registry/github/kpsr-core:Noetic_1.0.0
+      credentials:
+        username: AWS
+        password: ${{ secrets.ECR_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Build
+        run:
+          mkdir build && cd build && cmake -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false \
```

Comment:
Why the ROS builds do not end in error? They all seem to pass, and they should fail as the action does not finish. Is it because they finish before getting to a point of failure?